config/routes.rb

```diff
@@ -1,3 +1,33 @@
 Rails.application.routes.draw do
+  # Routes for signing up
+
+  match("/leader_sign_up", { :controller => "leader_sessions", :action => "registration_form", :via => "get"})
+  
+  # Routes for signing in
+  match("/leader_sign_in", { :controller => "leader_sessions", :action => "session_form", :via => "get"})
+  
+  match("/leader_verify_credentials", { :controller => "leader_sessions", :action => "add_cookie", :via => "post"})
+  
+  # Route for signing out
+  
+  match("/leader_sign_out", { :controller => "leader_sessions", :action => "remove_cookies", :via => "get"})
+  
+  # Route for creating account into database 
+
+  match("/insert_leader", { :controller => "leaders", :action => "create", :via => "post" })
+  
+  # Route for editing account
+  
+  match("/edit_leader", { :controller => "leaders", :action => "edit_account", :via => "get"})
+  
+  match("/update_leader", { :controller => "leaders", :action => "update_account", :via => "post"})
```

Comment:
What do you think about making the table-CRUD parts of `draft:account` match whatever naming conventions we use in `draft:resource`, for the subset of actions that exist for both (`update`, `create`, `destroy`, etc)? E.g. path names, action names.